plugins/e57/io/Scan.cpp

```diff
@@ -59,17 +59,6 @@ e57::CompressedVectorNode Scan::getPoints() const
     return *m_rawPoints;
 }
 
-bool Scan::getLimits(pdal::Dimension::Id pdalId, std::pair<double, double>& minMax) const
-{
-    auto itMinMax = m_valueBounds.find(pdalId);
-    if (itMinMax != m_valueBounds.end())
-    {
-        minMax = itMinMax->second;
-        return true;
-    }
-
-    return false;
-}
 
 bool Scan::hasPose() const
```

Comment:
Don't you just want to call this one time?  Or one time per chunk?  It seems like it's getting called for every point.  Can you just check m_hasPose at the beginning and exit if you've already checked or something like that?

Reply:
This needs to be called once per e57 scan.
We can do the following : 

- Call this once a new scan is setup and capture status in `E57Reader::m_needsTransform` (new member) and instead making call to `hasPose()`, Can just check `m_needsTransform` for every point.

-  Just make hasPose inline.

What do you think?